File: JokenpoClient/src/app/shared/modelos/JogadaResponseDTO.ts
export interface JogadaResponseDto {
    escolhaComputador: string,
    escolhaJogador: string,
    resultado: string;
    round: number;
    partida: number;

    partidaWinDraw: PartidaWinLossDraw

}


export interface PartidaWinLossDraw {
    winCount: number;
    lossCount: number;
    drawCount: number;

}

File: API/Business/Dtos/JogadaResponseDto.cs
using Business.Modelos;

namespace Business.Dtos
{
    public class JogadaResponseDto
    {
        public int round { get; set; }
        public string resultado { get; set; }
        public string escolhaJogador { get; set; }
        public string escolhaComputador { get; set; }

        public PartidaWinLossDraw PartidaWinDraw{ get; set; }
        public int partida { get; set; }
    }
}